src/main/java/org/gridsuite/network/map/model/VoltageLevelMapData.java

```diff
@@ -32,10 +33,23 @@ public class VoltageLevelMapData {
 
     private double nominalVoltage;
 
+    private double lowVoltageLimit;
```

Comment:
@JsonInclude(JsonInclude.Include.NON_NULL)
private Double lowVoltageLimit;

same for other double fields